playwright-e2e/authentication/auth-base.ts

```diff
@@ -14,6 +14,14 @@ export async function fillInEmailPassword(
 ): Promise<void> {
   const { email, password, waitForNavigation = true, waitForAuth = true } = opts;
 
+  await page.waitForSelector('.auth0-lock-header-welcome', { state: 'visible' });
+
+  const notYourAcctLink = page.locator('#auth0-lock-container-1 .auth0-lock-alternative-link:has-text("Not your account")');
+  try {
+    await notYourAcctLink.click({ timeout: 2000 });
+  } catch (err) {
+    // ignored
```

Comment:
should we put a custom error message here in case it fails?

Reply:
IMO, I don't think we need to log a message in catch block. I don't have have a strong preference. Let me know if you think we ought to log here.